libs/utils/package.json

```diff
@@ -1,6 +1,6 @@
 {
 	"name": "@studiohyperdrive/ngx-utils",
-	"version": "18.1.0",
+	"version": "18.1.1",
```

Comment:
Now just change it here as well and you're done :D

Reply:
DONE 🤦